### PR TITLE
tw placetype local and more

### PR DESCRIPTION
**Pull request by nvkelso**
Will fix a portion of https://github.com/whosonfirst-data/whosonfirst-data/issues/2154. Sets placetype local, names, populations, and statoids properties for CARTO

### CODE REVIEW

File: data/110/849/700/5/1108497005.geojson [hunk 1]
```diff
@@ -43,7 +49,7 @@
         "Landkreis Taichung"
     ],
     "name:eng_x_preferred":[
-        "Taichung County"
```
Comment (stepps00) [inline data/110/849/700/5/1108497005.geojson:46]: Should this name be stored as an alt?

File: data/110/849/700/5/1108497005.geojson [hunk 2]
```diff
@@ -135,10 +154,12 @@
     "wof:lang_x_spoken":[
         "zho"
     ],
-    "wof:lastmodified":1690875952,
+    "wof:lastmodified":1694493047,
     "wof:name":"Taichung",
     "wof:parent_id":85679617,
     "wof:placetype":"macrocounty",
+    "wof:population":2731056,
+    "wof:population_rank":18,
```
Comment (stepps00) [inline data/110/849/700/5/1108497005.geojson:162]: Nice!

File: data/110/849/986/1/1108499861.geojson [hunk 3]
```diff
@@ -58,7 +64,7 @@
         "\u039a\u03bf\u03bc\u03b7\u03c4\u03b5\u03af\u03b1 \u03a4\u03c3\u03b9\u03b1\u03b3\u03af"
     ],
     "name:eng_x_preferred":[
-        "Chiayi County"
```
Comment (stepps00) [inline data/110/849/986/1/1108499861.geojson:61]: Probably needs to be stored as an alt, no?

**Comment by stepps00** [comment]
A few comments about names being removed, but other than that.. LGTM

File: data/110/849/700/5/1108497005.geojson (hunk 1 above)
Comment (nvkelso) [reply data/110/849/700/5/1108497005.geojson:46]: Yes, it should. See details in other PR comment...

File: data/110/849/986/1/1108499861.geojson (hunk 3 above)
Comment (nvkelso) [reply data/110/849/986/1/1108499861.geojson:61]: Agree, I'll add a new commit...

Comment (nvkelso) [reply data/110/849/986/1/1108499861.geojson:61]: Actually these are a little hard for me to pull back out...

And generally these "County" values are now available in the `label:*` namespace so they are not removed completely, so I think it's OK?

For scope: In Taiwan this looks to affect 72 records.

**Pull request merged**